tests/test_i18n.py

```diff
@@ -51,6 +51,7 @@ def test_convert_language_code_input_validation(language_code, is_valid):
         ('django', True),
         ('modeltrans', True),
         ('next.js', True),
+        ('wagtail Locale', True),
```

Comment:
For some reason it really hurts my eyes that `Locale` starts with uppercase when wagtail does not. Since django is lowercase above, wagtail should be, but couldn't we think it's all lowercase then?

Reply:
I know Locale is the name of the model class and therefore uppercase, but I would still maybe prefer lowercase. But if you disagree, I'm fine with it.

Reply:
I was thinking that as well but also I'm referring explicitly to the `Locale` class in wagtail, so seemed wrong to write it lowercase given that it is a class name. I'm wondering if we can derive that that is the format used throughout wagtail and use simply the name `wagtail`?

Reply:
Yeah, that would be fine by me! Let's do that and if for some unfathomable reason we will need another wagtail language code convention, then let's call that 'wagtail something something'.

Reply:
@MWargelin Remember to rename this in the other PR too.

Reply:
Great, fixed now for this PR, I'll fix the other one now :+1: